README.md

```diff
@@ -26,6 +26,7 @@ for a Python package.
   - Support to pre-commit
   - CI with github actions
   - Release workflow with semantic release and github actions
+  - Finding common security problems in Python code using [bandit](https://bandit.readthedocs.io/en/latest/)
 
 ## Quickstart
```

Comment:
@xmnlab To generate a Python package project, using cookiecutter or cookieninja still works, but which one should we use? cookieninja?

Reply:
Yep cookieninja. Although it is not available on conda-forge yet, just pypi

Reply:
Thank you. I'm going to change the readme with this information, and then this PR will be ready for another review.